tests/assets/ansible/playbook_rook_ses_upstream.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
(nit) Would prefer this file to be renamed to avoid the confusion between ses and upstream. Perhaps `playbook_rook_ses_custom_image.yaml`?

Reply:
I am fine with that change, too. My intention was to use the naming-scheme which is already present (`playbook_rook_upstream.yaml`).